editoast/src/converters/generate_routes.rs

```diff
@@ -401,16 +379,16 @@ mod tests {
     }
 
     #[test]
-    /* ----o---d>---
-            \------
+    /* ----o---<d>---
```

Comment:
I think we can remove the brackets since detectors don't have directions.


```suggestion
    /* ----o----d----
```